pydis_site/apps/content/views/tags.py

```diff
@@ -0,0 +1,104 @@
+import re
+import typing
+
+import frontmatter
+import markdown
+from django.conf import settings
+from django.http import Http404
+from django.urls import reverse
+from django.views.generic import TemplateView
+
+from pydis_site.apps.content import utils
+from pydis_site.apps.content.models import Tag
+
+COMMAND_REGEX = re.compile(r"`*!tags? (?P<name>[\w\d-]+)`*")
```

Comment:
Tag groups are separated by spaces, rather than dashes.
```suggestion
COMMAND_REGEX = re.compile(r"`*!tags? (?P<name>[\w\d ]+)`*")
```

Reply:
Hmm, that dash is for tags, not tag groups, so I think it's still necessary (for instance `args-kwargs`). I hadn't considered tag groups at all when writing this, but I don't think they're separated by spaces either, I think the tag code does not support that. The relevant code for parsing tag commands is:

https://github.com/python-discord/bot/blob/dc9cb7910040f32d91aa546eec23b942a5a72203/bot/exts/info/tags.py#L66-L68


which leads me to believe that tags currently have to follow the `(tag-group)? tag-name` structure. I think it's simplest to write a regex which captures everything after `!tag` and handle the logic in the replacement function.

Reply:
Yea that seems reasonable, when i said separated by spaces I meant the group and the tag itself, I forgot tags could have hyphens in them too.

Reply:
Actually it might be pretty difficult to accurately discern what's a tag group, and what's a tag with a normal sentence after it (for instance this tag link at the bottom of [args-kwargs](https://deploy-preview-763--pydis-static.netlify.app/pages/tags/args-kwargs/) which has a sentence after it). The only reliable way would be to either make a request to the page (reverse does not validate kwargs), or to use the get_tag function, but that can make an extra request (commit data) if the tag is not part of a group, neither of which are ideal solutions.

Reply:
This regex achieves the first part of that:
```regex
`*!tags? (?P<first>[\w-]+)(?P<second> [\w-]+)?`*
```

It might optionally match a second word after the first one, and if it does, that could be a tag group combination

Reply:
Seems like it would do the job, obviously we'd need to update the tag model and the logic to get the url. So I'd be happy to merge this and have that as a separate PR if needed. Especially since we don't have any tag groups atm anyway.

Reply:
I wrote the code for it, that was pretty simple, but I'm having unrelated trouble getting the test suite running. I'll probably have the fix out tomorrow

Reply:
Fixed in 91495dac